File: GreenTicket-ClientApp/src/app/models/registerUserDto.ts
export interface RegisterUserDto {
    firstName: string,
    lastName: string,
    dateOfBirth: Date | null,
    email: string,
    password: string,
    repeatPassword: string,
    street: string,
    streetNo: string,
    postalCode: string,
    city: string,
    countryId: string
}

File: GreenTicket-WebAPI/Models/RegisterUserDto.cs
namespace GreenTicket_WebAPI.Models
{
    public class RegisterUserDto
    {
        public string Email { get; set; } = null!;
        public string Password { get; set; } = null!;
        public string RepeatPassword { get; set; } = null!;

        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public DateTime DateOfBirth { get; set; } 
        public string Street { get; set; } = null!;
        public string StreetNo { get; set; } = null!;
        public string PostalCode { get; set; } = null!;
        public string City { get; set; } = null!;
        public string CountryId { get; set; } = null!;
    }
}
